### PR TITLE
#1770 | PatrolTester: added longPress gesture

### PR DESCRIPTION
**Pull request by Tamerlanchiques**
leancodepl#1770

### CODE REVIEW

File: packages/patrol_finders/lib/src/custom_finders/patrol_tester.dart [hunk 1]
```diff
@@ -275,6 +275,55 @@ class PatrolTester {
...
+  /// long press on:
+  ///
+  /// ```dart
+  /// // long presses on the third TextButton widget
+  /// await $(TextButton).at(2).longPress();
+  /// ```
+  ///
+  /// After long press gesture this method automatically calls
+  /// [WidgetTester.pumpAndSettle]. If you want to disable this behavior,
+  /// set [andSettle] to false.
```
Comment (bartekpacia) [inline packages/patrol_finders/lib/src/custom_finders/patrol_tester.dart:297]:
```suggestion
  /// [WidgetTester.pumpAndSettle]. If you want to disable this behavior,
  /// set [settlePolicy] to [SettlePolicy.noSettle].
```

**Review by bartekpacia — COMMENTED** [review]
Thanks for contributing!

Please also add a simple test [in this file](https://github.com/leancodepl/patrol/blob/555609d24f03df3a1f42797ab53a91b643b5639c/packages/patrol_finders/test/patrol_finder_test.dart). You can take example from other tests in that file.

**Comment by bartekpacia** [comment]
OK, thank you for contribution. On a second thought this is a simple method forward and doesn't require tests so badly.

**Comment by Tamerlanchiques** [comment]
Oh I thought to add tests at weekends but spend time on IRL activities :) 
Will add test coverage soon…

**Pull request merged**